CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
+## 2.1.0 (April 2, 2021)
+
+### ares-device
+* Supports screen capture using the `--capture-screen` option.
+
+### ares-generate
+* Updated the template of appinfo.json to apply new ACG groups.
+* Added '/' to config method in js_service templates.
+
+### ares-shell
+* Fixed an issue that environment variables using the `ares-shell -r` command were different from environment variables of the target device.
+
+## 2.0.3 (January 22, 2021)
+
+### ares
+* Updated help message.
+
+### Markdown documentation
+* Updated the README file.
+
 ## 2.0.2 (December 29, 2020)
-- Support Node v14.15.1.
+* Supports Node v14.15.1.
```

Comment:
If there is only one item, it seems to be better not to *.
It was decided by discussing it together before.

Reply:
I had discussion with @ssuminahn , and decided to keep *